node-lib/src/options.rs

```diff
@@ -121,12 +122,29 @@ pub struct RunOptions {
     pub p2p_disable_noise: Option<bool>,
 
     /// Optional list of boot node addresses to connect.
-    #[clap(long, value_name = "NODE")]
-    pub p2p_boot_node: Option<Vec<IpOrSocketAddress>>,
+    /// Can be specified multiple times and/or be a comma-separated list.
+    #[clap(
+        long,
+        value_name = "NODE",
+        env = "MINTLAYER_P2P_BOOT_NODE",
+        // Note: comma is the default separator, but we have to specify it explicitly,
+        // because the vec is wrapped in Option.
+        // FIXME: get rid of Option?
```

Comment:
No the option means that not specifying it is OK. Everything here should be an option. Am I missing something?

Reply:
> No the option means that not specifying it is OK. Everything here should be an option. Am I missing something?

Well, it's vector and zero length would mean the absence of a value too.

Reply:
Actually an empty vector to being equivalent to no value is a bug in clap, I'd say. A None should mean that the value isn't present in the arguments. An empty vector should mean that it was present and was selected to be empty. But if it's like that, then sure, you can remove the option. Though keeping the symmetry is a nice thing.